### DISCUSSION TITLE
[@next/third-parties/google] Support for additional configuration when initializing Google Analytics

### DISCUSSION THREAD

**Discussion by kkak10** [vercel/next.js#61560] [Ideas]
### Goals

Use the @next/third-parties/google package to provide additional options when initializing Google Analytics to expand the availability of other features that Google Analytics offers.

### Background

I tried to use Google Analytics's debug mode. However, currently, it is not possible to receive additional options when initializing Google Analytics. I want to expand the interface to add options including debug mode and others.

https://support.google.com/analytics/answer/7201382?hl=en#zippy=%2Cgoogle-tag-gtagjs

### Proposal

I'm planning to work on this soon and submit a PR for it. 

I'd like to hear opinions regarding the implementation of this feature.

**Comment by erwannbst** [comment]
Same need, I created a PR for this #63405

**Reply by techmangreat** [reply]
Can you PLEASE inform other people how to use the pull request that you merged? What use to OTHER people is it if you add something that only you can use? Where is the documentation? What is the parameter name? What kind of values does it accept? How do I turn on debug mode?
